--- binutils-2.19-dist/gas/config/tc-msp430.h	2007-07-03 04:01:04.000000000 -0700
+++ binutils-2.19/gas/config/tc-msp430.h	2009-10-13 02:33:35.000000000 -0700
@@ -99,6 +99,7 @@
 
 #define LEX_DOLLAR 0
 /* MSP430 port does not use `$' as a logical line separator */
+#undef LEX_DOLLAR
 
 #define TC_IMPLICIT_LCOMM_ALIGNMENT(SIZE, P2VAR) (P2VAR) = 0
 /*   An `.lcomm' directive with no explicit alignment parameter will
